### DISCUSSION TITLE
Request for Unsuspended GitHub Account

### DISCUSSION THREAD

**Discussion by Shikimoriix** [community/community#77037] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Dear GitHub Support Team,

I hope this message finds you well. My name is K 𝕏 7, and my GitHub username is @ikx7a. I am writing to you regarding the recent suspension of my GitHub account.

If there have been any violations on my part, I sincerely apologize, and I am committed to rectifying any issues promptly. Could you kindly provide me with details regarding the reason for the account suspension so that I can take appropriate measures to address it?

Furthermore, I kindly request the reinstatement of my GitHub account. GitHub plays a crucial role in my work and collaborative projects, and I am eager to continue contributing positively to the community.

As part of the reinstatement process, I would also appreciate it if you could guide me on how to download or receive all repositories associated with my account. It is imperative for me to have access to this data for ongoing and future projects.

Here are my contact details for any further communication:

- Email: ikx7a@hotmail.com

I understand the importance of maintaining a safe and respectful community on GitHub, and I am committed to abiding by the platform's guidelines. I appreciate your prompt attention to this matter and look forward to a swift resolution.

Thank you for your understanding and assistance.

Best regards,

K 𝕏 7
Username: @ikx7a

**Comment by FindMalek** [comment]
## Response to: Request for Unsuspended GitHub Account #77037

### Category: Profile

---

Hello K 𝕏 7 (@ikx7a),

Thank you for reaching out regarding the suspension of your GitHub account. We understand the importance of GitHub in your work and collaborative projects, and we appreciate your prompt communication on this matter.

**Addressing Your Concerns:**

1. **Reason for Suspension:**
   - We will review the specifics of your account's suspension, checking for any potential violations of our community guidelines or terms of service.
   - Detailed information about the reason for the suspension will be provided to you after our review.

2. **Rectifying Issues:**
   - Your willingness to rectify any issues is commendable. We will guide you on the necessary steps to resolve these issues once identified.

3. **Account Reinstatement:**
   - We understand the urgency of regaining access to your account. The steps towards reinstating your account will be discussed post-review.

4. **Access to Repositories:**
   - Guidance on how to download or receive all repositories associated with your account will be provided once your account access is restored.

Please allow us some time to review your account and the circumstances surrounding its suspension. We will contact you via the provided email address (ikx7a@hotmail.com) with further details and instructions.

We appreciate your understanding and commitment to adhering to GitHub's guidelines. Your cooperation is crucial for a quick resolution.

Thank you for your patience and understanding. We look forward to resolving this matter and helping you get back to contributing to the GitHub community.

Best regards,

@FindMalek
GitHub Support Team

**Comment by Shikimoriix** [comment]
Dear GitHub Support Team,

I hope this message finds you well. It has been over a month, and my GitHub account, [@ikx7a](www.github.com/ikx7a), is still suspended. Unfortunately, I have not received any response regarding the reason for the suspension.

This suspension has led to the deletion of all my projects, causing significant disruption. I kindly request immediate assistance in resolving this matter and understanding the grounds for the account suspension. Your prompt response and support in recovering my projects would be greatly appreciated.

For your reference, my contact email is ikx7a@hotmail.com. I am eager to continue contributing to the GitHub community, and your assistance in this matter is crucial.

Thank you for your attention to this urgent issue.

Sincerely,
[ikx7a](www.github.com/ikx7a)

**Comment by CodeWizard0214** [comment]
Hello, @Shikimoriix 
How are you?
I faced similar issue with you
I can login at the git, but I can not code commit, push, pull
So I made `Reinstatement Request ticket ` on March, 5th but I didin't reply yet
when can I get reply from git support team?
Thank you
